src/Components/Sorting/Sorting.js

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import CHARACTER from "./SortingData";
+import SortingChar from "./SortingChar";
+import "./Sorting.scss";
+
+class Sorting extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            charList: [],
+            typeBool: false,
+            charBool: false,
+        };
+    }
+
+    componentDidMount() {
+        this.setState({
+            charList: CHARACTER,
+        });
+    }
+
```

Comment:
```suggestion
class Sorting extends React.Component {
    constructor() {
        super();
        this.state = {
            charList: CHARACTER,
            typeBool: false,
            charBool: false,
        };
    }
```
CDM에서 setState 할 필요가 없는 경우인 것 같습니다.